src/http.rs

```diff
@@ -100,6 +101,13 @@ async fn handle_pj(scheduler: Scheduler, req: Request<Body>) -> Result<Response<
     Ok(Response::new(Body::from(proposal_psbt)))
 }
 
+#[derive(Clone, Deserialize, Serialize, Debug)]
+pub struct ScheduleResponse {
+    bip21: String,
+    address: String,
+    quote: Option<Quote>,
+}
```

Comment:
This should really be strongly typed with `bitcoin::Amount`, `bitcoin::Amount`, `std::time::Duration` (I have NO clue this u32 duration is. ms? seconds? minutes? months? you get my point) and `bitcoin::Address` but this PR works and we need to ship. It's not a regression to our codebase but I know we do cleaner work than this.